### ISSUE TITLE
Accessibility][Bug] - Settings Generator component - No text alternative for settings generator back button

### ISSUE THREAD

**Issue by karolgromacki** [SAP/fundamental-ngx#13564]
### Describe your proposal

In the Settings dialog, with small dimensions the 'back' button icon is visible, but it lacks a text alternative. Since it is an icon-only button with no associated text or accessible name, screen reader users cannot understand its purpose. And there is no way of adding the alternative text to this 'back' button.

<img width="2236" height="1736" alt="Image" src="https://github.com/user-attachments/assets/f3d593cf-635e-43cd-8b05-e29bb01ecf4c" />

Expected Behavior: 
The back button should have a text alternative, like an aria-label or hidden text, so screen readers can announce its function 

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 20.1.4, Fundamental Library: 0.57.4

### Steps to Reproduce (for bugs)

1. Go to https://fundamental-ngx.netlify.app/#/platform/settings-generator
2. Open developer tool by pressing F12.
3. Toggle device toolbar
4. Add custom desktop device with dimensions 800px * 800px.
5. Select Appearance in settings generator example
6. Hover over 'back' button

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by mikerodonnell89** [comment]
Thanks, thought we had already implemented this.
